src/main/kotlin/io/sc3/peripherals/prints/PrintBlockEntity.kt

```diff
@@ -53,7 +53,9 @@ class PrintBlockEntity(
     val block = cachedState.block as? PrintBlock ?: return
 
     world.setBlockState(pos, cachedState.with(PrintBlock.on, !on), Block.NOTIFY_ALL)
-    world.playSound(null, pos, SoundEvents.BLOCK_LEVER_CLICK, SoundCategory.BLOCKS, 0.3f, if (on) 0.6f else 0.3f)
+    if (!data?.isQuiet!!) {
```

Comment:
Reviewing from mobile for now but I'm always a little suspicious of non-null assertions, especially right after a nullable expression! You could do this a few ways but probably this will be fine for now:

```suggestion
    if (data?.isQuiet != false) {
```

This will evaluate if isQuiet? is true or null

Reply:
Your fix actually ended up doing the opposite of what was intended 😞 671c8a82bbfba7fdcded822cba00c103b768a9f9

Reply:
Lol thanks